pkg/server/api_plot.go

```diff
@@ -357,7 +357,9 @@ func makePlotsResponse(plotSeries map[string][]plot.Series, plotReq *PlotRequest
 					plotItem.Scale(plot.Value(scale))
 				}
 
-				seriesOptions[seriesItem.Name] = seriesItem.Options
+				if len(seriesItem.Options) != 0 {
```

Comment:
I am not certain to have understood the purpose of this change. Could you elaborate on this please?